client/api_group.go

```diff
@@ -23,72 +23,76 @@ import (
-	// RenewGroupMember renew a list of group members and their expiration time
 	RenewGroupMember(ctx context.Context, groupOwnerAddr, groupName string, memberAddresses []string, opts types.RenewGroupMemberOption) (string, error)
-	// ListGroupMembers returns a list of members contained within the group specified by the group id, including those for which the user's expiration time has already elapsed
 	ListGroupMembers(ctx context.Context, groupID int64, opts types.GroupMembersPaginationOptions) (*types.GroupMembersResult, error)
-	// ListGroupsByAccount returns a list of all groups that the user has joined, including those for which the user's expiration time has already elapsed
-	// By default, the user is the sender. Other users can be set using the option
 	ListGroupsByAccount(ctx context.Context, opts types.GroupsPaginationOptions) (*types.GroupsResult, error)
-	// ListGroupsByOwner returns a list of groups owned by the specified user, including those for which the user's expiration time has already elapsed
-	// By default, the user is the sender. Other users can be set using the option
 	ListGroupsByOwner(ctx context.Context, opts types.GroupsOwnerPaginationOptions) (*types.GroupsResult, error)
 }
 
-// CreateGroup create a new group on greenfield chain, the group members can be initialized or not
+// CreateGroup - Create a new group without group members on Greenfield blockchain, and group members can be added by UpdateGroupMember transaction.
+//
+// ctx: Context variables for the current API call.
+//
+// groupName: The group name identifies the group.
+//
+// opt: The options for customizing a group and transaction.
+//
+// ret1: Transaction hash return from blockchain.
+//
+// ret3: Return error when the request failed, otherwise return nil.
 func (c *Client) CreateGroup(ctx context.Context, groupName string, opt types.CreateGroupOptions) (string, error) {
 	createGroupMsg := storageTypes.NewMsgCreateGroup(c.MustGetDefaultAccount().GetAddress(), groupName, opt.Extra)
 	return c.sendTxn(ctx, createGroupMsg, opt.TxOpts)
 }
 
-// DeleteGroup send DeleteGroup txn to greenfield chain and return txn hash
+// DeleteGroup - Delete a group on Greenfield blockchain.
+//
+// ctx: Context variables for the current API call.
+//
+// groupName: The group name identifies the group.
+//
+// opt: The options for customizing the transaction
+//
+// ret1: Transaction hash return from blockchain.
+//
+// ret3: Return error when the request failed, otherwise return nil.
 func (c *Client) DeleteGroup(ctx context.Context, groupName string, opt types.DeleteGroupOption) (string, error) {
 	deleteGroupMsg := storageTypes.NewMsgDeleteGroup(c.MustGetDefaultAccount().GetAddress(), groupName)
 	return c.sendTxn(ctx, deleteGroupMsg, opt.TxOpts)
 }
 
-// UpdateGroupMember support adding or removing members from the group and return the txn hash
+// UpdateGroupMember - Update a group by adding or removing members.
+//
```

Comment:
Who have the auth to update group member, need to add more details for this API

Reply:
added